Xamarin.Forms.Build.Tasks/Xamarin.Forms.Build.Tasks.csproj

```diff
@@ -8,7 +8,7 @@
 	<PropertyGroup>
 	    <_DisableStrongNamer Condition=" '$(_DisableStrongNamer)' != 'True' ">False</_DisableStrongNamer>
 	    <AssemblyName>Xamarin.Forms.Build.Tasks</AssemblyName>
-	    <TargetFramework>netstandard2.0</TargetFramework>
+	    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
```

Comment:
the order is important here, as vsmac doesn't support multitargetting yet, and see the first on as the single TargetFramework